test/integration/patients/worklist.js

```diff
@@ -45,12 +47,14 @@ context('worklist page', function() {
         fx.data[1].relationships.state = { data: { id: '55555' } };
         fx.data[1].attributes.name = 'Last In List';
         fx.data[1].attributes.updated_at = moment().subtract(2, 'days').format();
+        fx.data[1].attributes.due_date = moment().add(5, 'days').format();
```

Comment:
I think these'll be `moment(local)....format('YYYY-MM-DD')`  I don't think we should format these with a timezone.